core/src/main/java/com/radixdlt/bootstrap/RadixNodeBootstrapperModule.java

```diff
@@ -90,12 +92,19 @@ public RadixNodeBootstrapperModule(RuntimeProperties properties) {
   @Override
   public void configure() {
     bind(RuntimeProperties.class).toInstance(this.properties);
-    bind(Network.class).toInstance(readNetworkFromProperties(this.properties).unwrapRight());
+    final var network = readNetworkFromProperties(this.properties).unwrapRight();
+    bind(Network.class).toInstance(network);
+    bind(Addressing.class).toInstance(Addressing.ofNetwork(network));
     bind(GenesisFromPropertiesLoader.class).toInstance(new GenesisFromPropertiesLoader(properties));
     bind(RadixNodeBootstrapper.class).in(Scopes.SINGLETON);
     install(new MetricsModule());
     install(new CryptoModule());
     install(new NodeStorageLocationFromPropertiesModule());
+
+    // This isn't strictly required by the bootstrapper, but
+    // we're loading it up to verify that node's keystore
+    // configuration works.
```

Comment:
Can we explain that this will error at Guice instantiation if the key store isn't found?

Reply:
yup, will do in a follow up PR